test/math.jl

```diff
@@ -1,877 +1,934 @@
-@test zeta(27, 3.1) ≅ 5.413318813037879056337862215066960774064332961282599376e-14
-@test zeta(27, 2) ≅ 7.4507117898354294919810041706041194547190318825658299932e-9
-@test 1e-12 > relerr(zeta(27, -105.3), 1.3113726525492708826840989036205762823329453315093955e14)
-@test polygamma(4, -3.1+Inf*im) == polygamma(4, 3.1+Inf*im) == 0
-@test polygamma(4, -0.0) == Inf == -polygamma(4, +0.0)
-@test zeta(4, +0.0) == zeta(4, -0.0) ≅ pi^4 / 90
-@test zeta(5, +0.0) == zeta(5, -0.0) ≅ 1.036927755143369926331365486457034168057080919501912811974
-@test zeta(Inf, 1.) == 1
-@test zeta(Inf, 2.) == 0
-@test isnan(zeta(NaN, 1.))
-@test isa([digamma(x) for x in [1.0]], Vector{Float64})
-@test isa([trigamma(x) for x in [1.0]], Vector{Float64})
-@test isa([polygamma(3,x) for x in [1.0]], Vector{Float64})
-@test zeta(2 + 1im, -1.1) ≅ zeta(2 + 1im, -1.1+0im) ≅ -64.580137707692178058665068045847533319237536295165484548 + 73.992688148809018073371913557697318846844796582012921247im
-@test polygamma(3,5) ≈ polygamma(3,5.)
-
-@test zeta(-3.0, 7.0) ≅ -52919/120
-@test zeta(-3.0, -7.0) ≅ 94081/120
-@test zeta(-3.1, 7.2) ≅ -587.457736596403704429103489502662574345388906240906317350719
-@test zeta(-3.1, -7.2) ≅ 1042.167459863862249173444363794330893294733001752715542569576
-@test zeta(-3.1, 7.0) ≅ -518.431785723446831868686653718848680989961581500352503093748
-@test zeta(-3.1, -7.0) ≅ 935.1284612957581823462429983411337864448020149908884596048161
-@test zeta(-3.1-0.1im, 7.2) ≅ -579.29752287650299181119859268736614017824853865655709516268 - 96.551907752211554484321948972741033127192063648337407683877im
-@test zeta(-3.1-0.1im, -7.2) ≅ 1025.17607931184231774568797674684390615895201417983173984531 + 185.732454778663400767583204948796029540252923367115805842138im
-@test zeta(-3.1-0.1im, 7.2 + 0.1im) ≅ -571.66133526455569807299410569274606007165253039948889085762 - 131.86744836357808604785415199791875369679879576524477540653im
-@test zeta(-3.1-0.1im, -7.2 + 0.1im) ≅ 1035.35760409421020754141207226034191979220047873089445768189 + 130.905870774271320475424492384335798304480814695778053731045im
-@test zeta(-3.1-0.1im, -7.0 + 0.1im) ≅ 929.546530292101383210555114424269079830017210969572819344670 + 113.646687807533854478778193456684618838875194573742062527301im
-@test zeta(-3.1, 7.2 + 0.1im) ≅ -586.61801005507638387063781112254388285799318636946559637115 - 36.148831292706044180986261734913443701649622026758378669700im
-@test zeta(-3.1, -7.2 + 0.1im) ≅ 1041.04241628770682295952302478199641560576378326778432301623 - 55.7154858634145071137760301929537184886497572057171143541058im
-@test zeta(-13.4, 4.1) ≅ -3.860040842156185186414774125656116135638705768861917e6
-@test zeta(3.2, -4) ≅ 2.317164896026427640718298719837102378116771112128525719078
-@test zeta(3.2, 0) ≅ 1.166773370984467020452550350896512026772734054324169010977
-@test zeta(-3.2+0.1im, 0.0) ≅ zeta(-3.2+0.1im, 0.0+0im) ≅ 0.0070547946138977701155565365569392198424378109226519905493 + 0.000768918217924305877455352854524969142390140505
```

Comment:
sorry this is supposed to be about cis not zeta, must've posted this on the wrong line